cns/restserver/util.go

```diff
@@ -164,8 +164,7 @@ func (service *HTTPRestService) saveNetworkContainerGoalState(req cns.CreateNetw
 		case cns.AzureFirstParty:
 			fallthrough
 		case cns.WebApps: // todo: Is WebApps an OrchastratorType or ContainerType?
-			var podInfo cns.KubernetesPodInfo
```

Comment:
Did you also validate ACI and WebApps scenario to ensure there is no breaking change there?